tests/v3_api/common.py

```diff
@@ -1102,3 +1104,66 @@ def validate_file_content(pod, content, filename):
     cmd_get_content = "/bin/bash -c 'cat {0}' ".format(filename)
     output = kubectl_pod_exec(pod, cmd_get_content)
     assert output.strip().decode('utf-8') == content
+
+def wait_for_mcapp_to_active(client, multiClusterApp, timeout=DEFAULT_MULTI_CLUSTER_APP_TIMEOUT):
+    time.sleep(5)
+    mcapps = client.list_multiClusterApp(uuid=multiClusterApp.uuid, name=multiClusterApp.name).data
+    start = time.time()
+    assert len(mcapps) == 1
+    mapp = mcapps[0]
+    while mapp.state != "active":
+        if time.time() - start > timeout:
+            raise AssertionError(
+                "Timed out waiting for state to get to active")
+        time.sleep(.5)
+        multiclusterapps = client.list_multiClusterApp(uuid=multiClusterApp.uuid, name=multiClusterApp.name).data
+        assert len(multiclusterapps) == 1
+        mapp = multiclusterapps[0]
+    return mapp
+
+
+def validate_mcapp_cluster(app_id, p_client):
+    mcapp = p_client.list_app(name=app_id).data
+    assert len(mcapp) == 1
+    app = mcapp[0]
+    return app
+
+
+def wait_for_mcapp_cluster_level_to_active(client, app_id, timeout=DEFAULT_MULTI_CLUSTER_APP_TIMEOUT):
+    mcapps = client.list_app(name=app_id).data
+    start = time.time()
+    assert len(mcapps) == 1
+    mapp = mcapps[0]
+    while mapp.state != "active":
+        if time.time() - start > timeout:
+            raise AssertionError(
+                "Timed out waiting for state to get to active")
+        time.sleep(.5)
+        apps = client.list_app(name=app_id).data
+        assert len(apps) == 1
+        mapp = apps[0]
+    return mapp
+
+
```

Comment:
This method can be moved to test_multi_cluster_app.py

Reply:
moved to the file.